python/iam-mfa.py

```diff
@@ -20,8 +20,12 @@ def evaluate_compliance(configuration_item):
         return "NOT_APPLICABLE"
 
     user_name = configuration_item["configuration"]["userName"]
-
+    
     iam = boto3.client("iam")
+    try:
+        iam.get_login_profile(UserName=user_name)
+    except:
```

Comment:
except ClientError as e:
     if 'NoSuchEntity' in e.response['Error']['Code']:
        return "COMPLAINT"
     else:
        raise e